bot-components/GitHub_queries.mli

```diff
@@ -0,0 +1,74 @@
+val pull_request_milestone_and_cards :
+     bot_info:Bot_info.bot_info
+  -> owner:string
+  -> repo:string
+  -> number:int
+  -> ( GitHub_types.project_card list * GitHub_GraphQL.milestone option
```

Comment:
Here we still use a type coming from `GitHub_GraphQL`. I don't know how this can work since this module is supposed to be private, but the type definitions from `GitHub_GraphQL` should be moved to `GitHub_types`.